Assets/HoloToolkit/Input/Scripts/SelectedObjectMessageSender.cs

```diff
@@ -0,0 +1,46 @@
+﻿using HoloToolkit.Unity;
+using UnityEngine;
+
+/// <summary>
+/// SelectedObjectMessageSender class sends a Unity message to currently selected object.
+/// Object selection is controlled via OnSelectObject and OnClearSelection events.
+/// Object selection and messages can be triggered using keyword responses registered in KeywordManager.
+/// </summary>
+public class SelectedObjectMessageSender : MonoBehaviour
```

Comment:
I guess clarifying here that this selection happens with a voice command Vs an air-tap gesture would be beneficial.
1. Is that true? 
2. Should this work in conjunction with air-tap?
3. If you have air-tap wired up then just saying 'Select' also triggers the same action.